packages/appstore/src/components/wallet-header/wallet-header.scss

```
@@ -0,0 +1,94 @@
+.wallet-header {
+    padding: 2.4rem;
+    background-color: var(--general-main-1);
+    border-radius: $BORDER_RADIUS * 4;
+    height: 12.8rem;
+
+    &__demo {
+        position: relative;
+        background-color: var(--wallet-demo-bg-color);
+    }
+    &__demo:before {
+        content: ' ';
+        display: block;
+        position: absolute;
+        left: 0;
+        top: 0;
+        width: 100%;
+        height: 100%;
```

Correct me if I'm wrong here but:

```suggestion
        position: absolute;
        inset: 0;
```